frontend/packages/knative-plugin/src/utils/get-knative-resources.ts

```diff
@@ -3,6 +3,7 @@ import { WatchK8sResources } from '@console/dynamic-plugin-sdk';
 import { FirehoseResource } from '@console/internal/components/utils';
 import { K8sResourceKind, PodKind, referenceForModel } from '@console/internal/module/k8s';
 import { KafkaConnectionModel } from '@console/rhoas-plugin/src/models';
+import { CamelKameletModel, GLOBAL_OPERATOR_NS } from '..';
```

Comment:
we would need an import as below, this would be causing cyclic dep i think

```suggestion
import { CamelKameletModel } from '../models';
import { GLOBAL_OPERATOR_NS } from '../const'
```